common/djangoapps/enrollment/data.py

```diff
@@ -221,6 +222,21 @@ def get_enrollment_attributes(user_id, course_id):
     return CourseEnrollmentAttribute.get_enrollment_attributes(enrollment)
 
 
+def unenroll_user_from_all_courses(user_id):
+    """
+    Set all of a user's enrollments to inactive.
+    :param user_id: The user being unenrolled.
+    :return: A list of all courses from which the user was unenrolled.
+    """
+    user = _get_user(user_id)
+    enrollments = CourseEnrollment.objects.filter(user=user)
+    with transaction.atomic():
+        for enrollment in enrollments:
+            _update_enrollment(enrollment, is_active=False)
```

Comment:
Are we "forgetting users" by deactivating their enrollments? Should we anonymize or delete the user's entries in the database?

Reply:
The plan for this endpoint is, I believe, solely to unenroll them by setting `is_active` to false. This is an early step in a several-step process, and a later step will handle anonymizing the user's ID and any PII in the enrollments table.

Reply:
Makes sense. Thanks for the explanation.